### ISSUE TITLE
Can not install on win 8.1

### ISSUE THREAD

**Issue by kerwitz** [justinlatimer/node-midi#60]
I am trying to install the node-midi package on Windows 8.
Python and Visual Studio Express 2013 have been installed beforehand but I keep getting this error:

    C:\Program Files (x86)\Windows Kits\8.1\Include\um\winsock2.h(882): fatal error C1083: Datei (Include) kann nicht geöffnet werden: "qos.h": No such file or directory [C:\Users\Marco\AppData\Roaming\npm\node_modules\midi\build\midi.vcxproj]

I can't find any information about the "gos.h" file that is missing and can't check the midi.vcxproj file because the installer removes it immediately as it seems.

Any hints on what I could try?

Thansk in advance.

**Comment by justinlatimer** [comment]
Did you eventually get this working? Or shall I close the issue?

**Comment by denoir** [comment]
I gave up on it as I did not have any more time to mess with this. I might give it another try next month but it really is a big dependency mess. As said above these problems are related to all C++ node modules on Windows 8 and therefor its a bug of [node-gyp](https://github.com/TooTallNate/node-gyp) as far as I can tell at this moment. Feel free to close this issue - I might come back with more information once I have any.
